apps/api/src/lesson/adminLesson.repository.ts

```diff
@@ -270,34 +510,34 @@ export class AdminLessonRepository {
   //   );
   // }
 
-  // async updateLessonItemDisplayOrder(chapterId: UUIDType, lessonId: UUIDType) {
-  //   await this.db.transaction(async (trx) => {
-  //     await trx.execute(sql`
-  //       UPDATE ${lessonItems}
-  //       SET display_order = display_order - 1
-  //       WHERE lesson_id = ${chapterId}
-  //         AND display_order > (
-  //           SELECT display_order
-  //           FROM ${lessonItems}
-  //           WHERE lesson_id = ${chapterId}
-  //             AND lesson_item_id = ${lessonId}
-  //         )
-  //     `);
-
-  //     await trx.execute(sql`
-  //       WITH ranked_lesson_items AS (
-  //         SELECT lesson_item_id, row_number() OVER (ORDER BY display_order) AS new_display_order
-  //         FROM ${lessonItems}
-  //         WHERE lesson_id = ${chapterId}
-  //       )
-  //       UPDATE ${lessonItems} li
-  //       SET display_order = rl.new_display_order
-  //       FROM ranked_lesson_items rl
-  //       WHERE li.lesson_item_id = rl.lesson_item_id
-  //         AND li.lesson_id = ${chapterId}
-  //     `);
-  //   });
-  // }
+  async updateLessonItemDisplayOrder(chapterId: UUIDType, lessonId: UUIDType) {
+    await this.db.transaction(async (trx) => {
+      await trx.execute(sql`
```

Comment:
in apps/api/src/chapter/repositories/adminChapter.repository.ts:98 we have example

Reply:
yes but one concerns lessons and the other chapters, it is a completely separate logic and we should not mix it so that it can be easier to manage in the future

Reply:
Both function in this case updates lessons, I talk about chapter approach as template